tests/set-working-directory_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"testing"
+
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/schema"
+	"github.com/appc/acbuild/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+)
+
+func TestSetWorkingDir(t *testing.T) {
+	workingDir := setUpTest(t)
+	defer cleanUpTest(workingDir)
+
+	const wd = "/root"
+
+	_, _, _, err := runACBuild(workingDir, "set-working-dir", wd)
+	if err != nil {
+		t.Fatalf("%v\n", err)
+	}
+	man := schema.ImageManifest{
```

Comment:
That would require either undoing any changes I made to `emptyManifestWithApp` or finding a way to deep copy the struct. I'll look into it.